job-utilities/local-development/README.MD

```diff
@@ -0,0 +1,13 @@
+# local development job-utilities
+
+Python utilities to help with local development. Currently there is just a program for scheduling jobs to run locally
+
+## local_scheduler
+
+This program takes the job details from the job-details.json file and maps them to a schedule using the schedule Python library.
+Cron jobs are assumed to be run once every day at a given time. Most jobs are likely to be interval based, which run every given period of time.
+
+### Running
+
+Ensure you have the right dependencies by running `pip install -r requirements.txt` and then simply run the local_scheduler.py program.
+Note that the program currently does not run jobs immediately upon scheduling.
```

Comment:
Does the CMR run jobs immediately now? I thought at least in the development environment there is a configurable 3 hour delay.  Do the jobs in SIT,UAT, Prod all run when the CMR starts up?

Reply:
I'm not sure how exactly it works locally, but in my experience when CMR is deployed to an environment there are definitely jobs that run on startup.